packages/nimble-components/.storybook/manager.js

```diff
@@ -1,4 +1,4 @@
-import { addons } from '@storybook/addons';
+import { addons } from '@storybook/manager-api';
 import theme from './theme';
 
```

Comment:
In this file below we probably need to update `item.title.startsWith('patterns/');` to `item.title.startsWith('Patterns/');`